indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -124,7 +124,9 @@ public TaskStatus call()
                         final File attemptDir = new File(taskDir, attemptUUID);
 
                         final ProcessHolder processHolder;
-                        final int childPort = portFinder.findUnusedPort();
+                        final int childQueryPort = portFinder.findUnusedPort();
```

Comment:
I do not have much experience with debugging web services...just curious can you please explain how it can be useful in debugging ?